### PR TITLE
test for create issue

### PR DESCRIPTION
**Pull request by liuliu-dev**

### CODE REVIEW

File: cypress/integration/utils/sharedTests/testDocs.ts [hunk 1]
```diff
@@ -68,6 +77,46 @@ const mergingAndDeletingBranch = (
 ];
 
 export const testDocs: Tests = [
+  //! CREATE A NEW ISSUE
+  newExpectationWithClickFlows(
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/testDocs.ts:81]: So this file is specifically testing the docs workflow, which is a separate workflow from creating an issue. All this issue code should go in a new file and imported in `newDatabaseWithEdits.spec.ts`. You shouldn't have any changes in this file

**Review by tbantle22 — COMMENTED** [review]
This is a great start! Besides my one comment, I think there could be a few more tests here:
- Clicking on issue and making sure state is correct (issue is comment)
- Edit issue title/description and making sure change saves
- Commenting on issue and updating comment
- Testing closing the issue and state updates
- Testing reopening issue and state updates

File: cypress/integration/utils/sharedTests/testDocs.ts (hunk 1 above)
Comment (liuliu-dev) [reply cypress/integration/utils/sharedTests/testDocs.ts]: added more tests, and created a new file for issue test.

**Review by tbantle22 — APPROVED** [review]
Great job! Just one small comment

File: cypress/integration/utils/sharedTests/testIssues.ts [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+const issueContent = "test content";
+const issueTitleEdits = "test edited";
+const issueContentEdits = "test content edited";
+const issueComment = "test comment";
+const issueCommentUpdate = "test comment update";
+
+export const typingExpectation = (
+  value: string,
+  datacy: string,
+  status?: string,
```
[the remark this answers is not in the document]
Comment (liuliu-dev) [reply cypress/integration/utils/sharedTests/testIssues.ts]: removed

**Pull request merged**